requirements.txt

```diff
@@ -49,12 +49,6 @@ gevent>=1.3.6
 # Used by semantic parsing code to strip diacritics from unicode strings.
 unidecode
```

Comment:
I don't think we need this unidecode library apart from for the semparse code

Reply:
This one could have been removed when we got rid of the SEMPRE integration, looks like.  Removing it now.